src/pages/task/actionButtons/relevantCommits/RelevantCommits.test.tsx

```diff
@@ -9,23 +9,23 @@ import {
 import { BASE_VERSION_AND_TASK, LAST_MAINLINE_COMMIT } from "gql/queries";
 import { renderWithRouterMatch, screen, userEvent, waitFor } from "test_utils";
 import { ApolloMock } from "types/gql";
-import { PreviousCommits } from ".";
+import { RelevantCommits } from ".";
 
-describe("previous commits", () => {
+describe("relevant commits", () => {
```

Comment:
I added to the existing tests instead of a new one and added some data! I think it makes sense to test it like the others but I am new to testing and if you feel otherwise I can switch it up